### ISSUE TITLE
Microsoft stop supporting Firebase.iOS

### ISSUE THREAD

**Issue by MateuszKapusta** [marcojak/MauiMTAdmob#89]
Microsoft stop supporting Firebase.iOS (https://github.com/dotnet/maui/discussions/20359) and now best option is to use (https://github.com/AdamEssenmacher/GoogleApisForiOSComponents). 

MauiMTAdmob uses Xamarin.Firebase.iOS.Core 8.10.0.3, so would you consider to update it to AdamE.Firebase.iOS.Core 10.24.0.2 in the future?

I got this error and only fix for now is to install original Xamarin.Firebase.iOS libraries. 

##[error]/Users/runner/hostedtoolcache/dotnet/packs/Microsoft.iOS.Sdk.net8.0_17.5/17.5.8018/targets/Xamarin.Shared.Sdk.targets(1641,3): Error : clang++ exited with code 1:
Undefined symbols for architecture x86_64:
  "_APMAnalyticsConfiguration", referenced from:
      +[FIRAnalytics startWithConfiguration:options:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMAppMeasurementOriginFirebase", referenced from:
      +[FIRAnalytics startWithConfiguration:options:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMConsentSettings3P", referenced from:
      +[FIRAnalytics setConsent:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMFormattedEventName", referenced from:
      +[FIRAnalytics logEventWithOrigin:name:parameters:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMFormattedUserPropertyName", referenced from:
      +[FIRAnalytics setUserPropertyString:forName:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMIsAnalyticsCollectionDeactivated", referenced from:
      +[FIRAnalytics startWithConfiguration:options:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMIsAnalyticsCollectionEnabled", referenced from:
      +[FIRAnalytics startWithConfiguration:options:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMMonitorLogTagOptionKey", referenced from:
      +[FIRAnalytics startWithConfiguration:options:] in FirebaseAnalytics(FIRAnalytics.o)
  "_APMSafelistedEventsOptionKey", referenced from:
      +[FIRAnalytics startWithConfiguration:options:] in FirebaseAnalytics(FIRAnalytics.o)
  "_OBJC_CLASS_$_APMAdExposureReporter", referenced from:
      _OBJC_CLASS_$_FIRAAdExposureReporter in FirebaseAnalytics(FIRAAdExposureReporter.o)
  "_OBJC_CLASS_$_APMAnalytics", referenced from:
      objc-class-ref in FirebaseAnalytics(FIRAnalytics.o)
  "_OBJC_CLASS_$_APMConditionalUserProperty", referenced from:
      _OBJC_CLASS_$_FIRAConditionalUserProperty in FirebaseAnalytics(FIRAConditionalUserProperty.o)

**Comment by marcojak** [comment]
Yes, as this seems the most promising package to replace the deprecated
libraries, I'll test everything and then if everything works fine I'll
switch to it.

**Comment by MateuszKapusta** [comment]
Can you tell me how long this might take?
